cmd/dockerd/daemon.go

```diff
@@ -653,6 +654,10 @@ func (cli *DaemonCli) getContainerdDaemonOpts() ([]supervisor.DaemonOpt, error)
 		opts = append(opts, supervisor.WithLogLevel(cli.LogLevel))
 	}
 
+	if logFormat := cli.Config.LogFormat; logFormat != "" && logFormat != "text" {
+		opts = append(opts, supervisor.WithLogFormat(logFormat))
+	}
+
```

Comment:
I'm a bit on the fence whether we should punch this through to containerd (related: https://github.com/moby/moby/pull/43946 and https://github.com/moby/moby/pull/43950)

Reply:
Without punching it through, I got inconsistent formatted log lines (some JSON, some text) when testing (since the dockerd logs also include the containerd logs).